HelloWeb/README.md

```diff
@@ -0,0 +1,29 @@
+# HelloWeb Application
+
+Gluon Application on the browser
+
+## Basic Requirements
+
+A list of the basic requirements can be found online in the [Gluon documentation](https://docs.gluonhq.com/#_requirements).
+
```

Comment:
Although we don't use GraalVM for this atm, the compile step fails if GraalVM is not installed. Either we should remove the restriction in Gluon Substrate, or document here that GraalVM is needed (for those people who don't follow links)?

Reply:
`nativerun` which also has no dependency fails if `GRAALVM_HOME` is not defined. We need to file issues and take care of it in the plugin instead.